src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/Models/ContainerServiceAgentPoolProfile.cs

```diff
@@ -45,8 +44,10 @@ public partial class ContainerServiceAgentPoolProfile
         /// 'Standard_DS4', 'Standard_DS11', 'Standard_DS12',
         /// 'Standard_DS13', 'Standard_DS14', 'Standard_GS1', 'Standard_GS2',
         /// 'Standard_GS3', 'Standard_GS4', 'Standard_GS5'</param>
-        /// <param name="fqdn">FDQN for the agent pool</param>
-        public ContainerServiceAgentPoolProfile(string name, string dnsPrefix, int? count = default(int?), string vmSize = default(string), string fqdn = default(string))
+        /// <param name="dnsPrefix">DNS prefix to be used to create the FQDN
+        /// for the agent pool.</param>
+        /// <param name="fqdn">FDQN for the agent pool.</param>
+        public ContainerServiceAgentPoolProfile(string name, int count, string vmSize, string dnsPrefix, string fqdn = default(string))
```

Comment:
@brendandburns this is a breaking change - the order of the parameters in the constructor has changed and default values for parameters have been removed